docker/docs/try-out-Athenz-with-self-signed-CA.md

```diff
@@ -0,0 +1,133 @@
+<a id="markdown-try-out-athenz-with-self-signed-ca-for-dev-only" name="try-out-athenz-with-self-signed-ca-for-dev-only"></a>
+# Try out Athenz with self-signed CA (for dev. ONLY)
+
+<!-- TOC -->
+
+- [Try out Athenz with self-signed CA (for dev. ONLY)](#try-out-athenz-with-self-signed-ca-for-dev-only)
+    - [Prerequisites](#prerequisites)
+    - [Prepare certificates](#prepare-certificates)
+    - [Overwrite env., and continue the setup](#overwrite-env-and-continue-the-setup)
+    - [Appendix](#appendix)
+        - [Note for mac users (not recommended)](#note-for-mac-users-not-recommended)
+
+<!-- /TOC -->
+
+<a id="markdown-prerequisites" name="prerequisites"></a>
+## Prerequisites
+
+All the setup commands below are expected to run inside [athenz-setup-env](../setup-scripts/Dockerfile) container.
+```bash
```

Comment:
I think it would be better to add the following command.
```
BASE_DIR="`git rev-parse --show-toplevel`"
```

Reply:
added in 392d4ea